src/transform/AstDumpDotPass.cpp

```diff
@@ -207,6 +216,49 @@ void AstDumpDotVisitor::visit( Specification& specification )
     m_stream << "}\n";
 }
 
+void AstDumpDotVisitor::visit( HeaderDefinition& node )
+{
+    DotLink link( this, &node );
+    dumpNode( node, "HeaderDefinition" );
+    RecursiveVisitor::visit( node );
+}
+
+void AstDumpDotVisitor::visit( InitDefinition& node )
+{
+    DotLink link( this, &node );
+    dumpNode( node, "InitDefinition" );
+    node.initToken()->accept( *this );
+    if( node.isSingleAgent() )
+    {
+        node.initPath()->accept( *this );
+    }
+    else
+    {
+        node.leftBraceToken()->accept( *this );
+        node.initializers()->accept( *this );
+        node.rightBraceToken()->accept( *this );
+    }
+}
+
+void AstDumpDotVisitor::visit( InitiallyDefinition& node )
+{
+    DotLink link( this, &node );
+    dumpNode( node, "InitiallyDefinition" );
+    RecursiveVisitor::visit( node );
+}
+
+void AstDumpDotVisitor::visit( InitializerDefinition& node )
+{
+    DotLink link( this, &node );
+    dumpNode( node, "InitializerDefinition" );
+    node.attributes()->accept( *this );
```

Comment:
Use `RecursiveVisitor::visit( node )` instead of the accepts. The dot dump should reflect the AST + recursive visitation as is.